lib/features/modeling/behavior/StickyBoundaryEventBehavior.js

```diff
@@ -0,0 +1,61 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import {
+  classes as svgClasses
+} from 'tiny-svg';
+
+
+var EXTENDED_HIT = 'djs-hit-extended';
+
+/**
+ * BPMN specific detach event behavior
+ */
+export default function StickyBoundaryEventBehavior(eventBus, elementRegistry) {
+
+  CommandInterceptor.call(this, eventBus);
+
+  var hostGfx = null;
```

Comment:
We should try to avoid caching stuff like that.

Remembering things in the drag context (event.context.***) is the preferred way to remember local state.